test/Constraints/generics.swift

```diff
@@ -578,3 +578,17 @@ func rdar39616039() {
   var c = foo(default: 42.0, ["foo": Float(0)])
   c += 1 // ok
 }
+
+// https://bugs.swift.org/browse/SR-8075
+
+func sr8075() {
```

Comment:
Please use two-space indent for consistency with the rest of the file. Also these days we generally put random crasher test cases into `validation-test/compiler_crashers_2_fixed/`

Reply:
This wasn't a crasher.